src/consts/warpRoutes.ts

```diff
@@ -1,70 +1,69 @@
-import { TokenConnectionType, TokenStandard, WarpCoreConfig } from '@hyperlane-xyz/sdk';
+import { WarpCoreConfig } from '@hyperlane-xyz/sdk';
 
 // A list of Warp Route token configs
 // These configs will be merged with the warp routes in the configured registry
 // The input here is typically the output of the Hyperlane CLI warp deploy command
 export const warpRouteConfigs: WarpCoreConfig = {
   tokens: [
     // TIA Celestia to Neutron
-    {
```

Comment:
Ah, I guess the manually defined routes here don't get filtered through the whitelist. Probably okay but we can change this if it's confusing.